### PR TITLE
Prep Broker for release

### PR DESCRIPTION
**Pull request by christothes**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by pvaneck — APPROVED** [review]

File: sdk/identity/Azure.Identity.Broker/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,6 @@
...
-## 1.1.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.1.0 (2024-04-09)
 
 ### Other Changes
```
Comment (g2vinay) [inline sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: This is technically a breaking change from last beta ?

**Review by g2vinay — APPROVED** [review]

File: sdk/identity/Azure.Identity.Broker/CHANGELOG.md (hunk 1 above)
Comment (christothes) [reply sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: yes, but I didn't think we categorize changes in betas as breaking changes.

**Review by jsquire — APPROVED** [review]

File: sdk/identity/Azure.Identity.Broker/CHANGELOG.md (hunk 1 above)
Comment (scottaddie) [reply sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: Correct. It's implied that we'll introduce breaks in Betas.

**Review by scottaddie — APPROVED** [review]

File: sdk/identity/Azure.Identity.Broker/CHANGELOG.md (hunk 1 above)
Comment (g2vinay) [reply sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: as a user, if I didn't try any beta versions, I see this change in the minor version bump. it comes across as a breaking change from last stable release.

Comment (christothes) [reply sdk/identity/Azure.Identity.Broker/CHANGELOG.md:6]: It's additive from the previous stable release.

**Pull request merged**